triton-server/config.ts

```diff
@@ -52,13 +54,19 @@ export default {
     mail: {
         errorMonitoring: TRITON_ENVIRONMENTS.ERROR_MONITORING_EMAIL,
         techSupport: TRITON_ENVIRONMENTS.TECH_SUPPORT_EMAIL,
+        toValidationNotification: TRITON_ENVIRONMENTS.TO_VALIDATION_EMAIL,
+        toTestEmail: TRITON_ENVIRONMENTS.TO_TEST_EMAIL,
     },
 
     sftp: {
         server: TRITON_ENVIRONMENTS.SFTP_SERVER,
         port: TRITON_ENVIRONMENTS.SFTP_PORT,
     },
 
+    email_testing: {
+        to: TRITON_ENVIRONMENTS.TO_TEST_EMAIL,
+    },
+
```

Comment:
redundant variable?

Reply:
i'll modify this in the next pr, to include local testing vars